strimzi/strimzi-kafka-operator
# ST for rolling update

### Type of change
- System tests

### Description
PR contains new ST to check ability to trigger a rolling update for Kafka and Zookeeper. 
Test case:
1. Deploy ephemeral Kafka cluster
2. Set annotation "strimzi.io/manual-rolling-update=true" for kafka statefulset 
3. Check annotation in the metadata
4. Wait when annotation will be removed 
5. Check log messages about triggering a rolling update
...
The same for Zookeeper


### Checklist
